ops/app/operables.nix

```diff
@@ -0,0 +1,12 @@
+let
+  inherit (inputs) std;
+  inherit (inputs.nixpkgs) lib;
+  # TODO: understand if CARDANO_NODE_CONFIG_PATH has appropriate defaults (depending on network)
+  # see also epic about organizing defaults in a single place
+in {
+  server = std.lib.ops.mkOperable {
+    package = cell.packages.server;
+    runtimeScript = "${cell.packages.server}/bin/cli";
```

Comment:
**NOT READY TO BE MERGED!**

Reply:
Yes, absolutely. The questions here is how do we inject `CARDANO_NODE_CONFIG_PATH` and the corresponding data?

Since it's sort of in the category a dockerfile's `ADD config.json` and quite a crucial source of configuraition data, this would best moved here for full operational transparency visibility.

Like on the other TODO, we don't need to expand the scope, but I'd like to get the discussion started with @rhyslbw and the team.

Reply:
> how do we inject CARDANO_NODE_CONFIG_PATH and the corresponding data?

Please elaborate, the issue is not clear to me.

Reply:
Please just export the variable. You can copy the logic from `wallet-world`

Reply:
Added in aa80281462df5fc6dc04448bd9471735e3cdcfb5. It's ok-ish. Conceptually, I'd rather like to inject these configuration invariants during runtime and also directly from the cardano _source of truth_ rather than a second or third order replica.

Let's see how we fare with this, it's not likely to cause any problems, and we can revisit later.